index.src

```diff
@@ -34,6 +34,10 @@
     <dd>
       A step-by-step guide to get up and running with a specification on GitHub, from Tobie.
     </dd>
+    <dt><a href="workflow.html">Workflow for editors and other contributors</a></dt>
+    <dd>
+      A guide for a clean and sane workflow for editors.
```

Comment:
Maybe s/sane/proven/ to avoid the implication that other ways of doing things couldn't work.